QuickOSM/core/actions.py

```diff
@@ -16,17 +23,41 @@
 ACTIONS_VISIBILITY = ['Canvas', 'Feature', 'Field']
 
 
-def add_actions(layer: QgsVectorLayer, keys: list):
+def add_actions(layer: QgsVectorLayer, keys: list, visibility: str = None, dialog: QDialog = None):
     """Add actions on layer.
 
     :param layer: The layer.
     :type layer: QgsVectorLayer
 
     :param keys: The list of keys in the layer.
     :type keys: list
+
+    :param visibility: Visibility for reload action.
+    :type visibility: str
+
+    :param dialog: Dialog for reload action.
+    :type dialog: QDialog
     """
     actions = layer.actions()
 
+    if visibility:
```

Comment:
Or make a new function ?
`add_action_reload_query` ? We won't need the "visiblity" string
Because you have a return just after.